### DISCUSSION TITLE
Seems like there's a bug with m4a playlist files

### DISCUSSION THREAD

**Discussion by stefanos82** [audacious-media-player/audacious#106] [Q&A] [answered]
Website https://nightride.fm uses https://nightride.fm/stream/nightride.m4a behind the scenes to stream music 24/7.

I have tried to play it with the latest repo commit and I got the following error messages:

![image](https://github.com/audacious-media-player/audacious/assets/1777466/a8ce0e1e-db9d-43f4-b536-d51dfb1cc619)

I have tested it with mpv and VideoLAN and both play it just fine.

Any feedback on this issue of mine?

**Comment by radioactiveman** [comment]
Feedback for now is that I can reproduce it. Not sure yet why it doesn't work with Audacious.

**Reply by stefanos82** [reply]
What caught my attention is that `moov atom`; did it mean `mov atom` perhaps and that typo is causing the problem?

**Reply by radioactiveman** [reply, marked as answer]
That's not the issue. The reason why the stream doesn't play is because avformat_open_input (provided by FFmpeg) fails.
And this is caused by a wrong file extension. nightride.fm uses incorrectly m4a as file extension but in fact streams MP3 content.

This is no issue in Audacious, so please report it to nightride.fm.
You may use https://nightride.fm/stream/nightride.ogg or https://nightride.fm/stream/nightride.mp3 until they fix it.

**Reply by stefanos82** [reply]
Thank you very much!
